src/components/Tabs/index.js

```diff
@@ -23,6 +24,9 @@ export default class Tabs extends React.Component {
   handleClick(index, event) {
     event.preventDefault();
     this.setState({ selected: index });
+    if (typeof this.props.onChange === 'function') {
+      this.props.onChange(index);
```

Comment:
I think the index is enough for a callback like this.  The currently visible component should be accessible via a different mechanism so that it can be queried.  I just want to be notified of a change.